### PR TITLE
Align lobby bits in the player tab in TD

### PR DESCRIPTION
**Pull request by dragunoff**
I noticed that some of the lobby bits are misaligned. Minor UI polish.

lobby-bits-td

### CODE REVIEW

**Review by abcdefg30 — DISMISSED** [review]

File: mods/cnc/chrome/lobby-players.yaml [hunk 1]
```diff
@@ -198,7 +198,6 @@ Container@LOBBY_PLAYER_BIN:
 							Template: ANONYMOUS_PLAYER_TOOLTIP
 						Label@NAME:
 							X: 39
-							Y: 0 - 1
```
Comment (pchote) [inline mods/cnc/chrome/lobby-players.yaml:201]: We'll need the same change in `TEMPLATE_EMPTY` too.

File: mods/cnc/chrome/lobby-players.yaml [hunk 2]
```diff
@@ -148,7 +148,7 @@ Container@LOBBY_PLAYER_BIN:
 							Height: 25
 							Font: Regular
 						Image@STATUS_IMAGE:
-							X: 521
+							X: 529
```
Comment (pchote) [inline mods/cnc/chrome/lobby-players.yaml:151]: I believe we'll need the same change in `TEMPLATE_EDITABLE_SPECTATOR` and `TEMPLATE_NONEDITABLE_SPECTATOR` too.

**Review by pchote — COMMENTED** [review]
LGTM, but i found a couple of additional inconsistencies that we should also fix:

**Comment by pchote** [comment]
Do the other mods have similar issues? It would be good to file a second PR fixing those for the playtest too.

**Comment by dragunoff** [comment]
> Do the other mods have similar issues? It would be good to file a second PR fixing those for the playtest too.

I checked the other mods before filing this PR and I didn't spot any issues. Should I file another PR against the prep branch?

**Comment by pchote** [comment]
Sorry, I should have added "if needed" to that sentence. I meant that any fixes, if needed, for the other mods didn't need to block merging this one.

**Comment by dragunoff** [comment]
Updated to address comments.

File: mods/cnc/chrome/lobby-players.yaml (hunk 1 above)
Comment (dragunoff) [reply mods/cnc/chrome/lobby-players.yaml:201]: In `TEMPLATE_NONEDITABLE_SPECTATOR` as well.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/a5d4c732f069f958052b4c880d22ba0808a9fc39...81449281c4645b825367c66a3dba79e0cba6847c

ecbcc9c

**Pull request merged**